build.sbt

```diff
@@ -3575,6 +3576,31 @@ lazy val `std-aws` = project
       "software.amazon.awssdk" % "sso"                     % awsJavaSdkV2Version,
       "software.amazon.awssdk" % "ssooidc"                 % awsJavaSdkV2Version
     ),
+    modulePath := {
+      val externalModIds = Seq(
```

Comment:
@Akirathan, why do we have to repeat all these modules when they are already in `libraryDependencies`?

Reply:
It is really simpler: cfdda7b